### ISSUE TITLE
ECONNRESET on blobClient.putFile

### ISSUE THREAD

**Issue by brollb** [webgme/webgme-engine#240]
Uploading files to the GME blob can sometimes result in ECONNRESET errors which has been causing downstream issues like https://github.com/deepforge-dev/deepforge/issues/1866

**Comment by brollb** [comment]
(I suspect this is due to the uploaded file being too large)

**Comment by brollb** [comment]
@kecso - Any updates/thoughts on this?

**Comment by hneema08** [comment]
@kecso Yes, I'm also seeing that GetCifarData operation succeeds in downloading the dataset but fails while uploading to the blob storage. Is this something easily fixable?

I guess to reproduce it you might try running the GitCifarData example or just create a pipeline with only that operation in it (with the latest deepforge repo.) -- choose "WebGME Blob storage" as the storage adapter in the pipeline execution.

**Comment by brollb** [comment]
It looks like this happens pretty consistently after 30 seconds so I suspect it is a request timeout setting somewhere...
